blocks/section-title-block/features/section-title/default.jsx

```diff
@@ -1,16 +1,55 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
-import GlobalContentSectionTitle from "./_children/global-content";
-import CustomContentSectionTitle from "./_children/custom-content";
+import { useAppContext } from "fusion:context";
+import { useContent } from "fusion:content";
+import { formatURL, Heading, Join, Link, Separator, Stack } from "@wpmedia/arc-themes-components";
 
-const SectionTitleContainer = ({
-	customFields: { inheritGlobalContent = true, sectionContentConfig } = {},
-} = {}) => {
-	if (inheritGlobalContent) {
-		return <GlobalContentSectionTitle />;
-	}
+const BLOCK_CLASS_NAME = "b-section-title";
 
-	return <CustomContentSectionTitle contentConfig={sectionContentConfig} />;
+export const SectionTitle = ({ content }) =>
+	content && (content.name || content.display_name) ? (
+		<Stack className={BLOCK_CLASS_NAME}>
+			<Heading>{content.name || content.display_name}</Heading>
+			<Stack direction="horizontal" wrap="wrap">
+				<Join separator={Separator}>
+					{!!(content.children && content.children.length > 0) &&
+						content.children.map((child) => {
+							if (child.node_type && child.node_type === "link") {
+								return (
+									<Link href={formatURL(child.url)} key={child.url}>
+										{child.display_name}
+									</Link>
+								);
+							}
+
+							if (child._id && child.name) {
+								return (
+									<Link href={formatURL(child._id)} key={child._id}>
+										{child.name}
+									</Link>
+								);
+							}
+							return null;
+						})}
+				</Join>
+			</Stack>
+		</Stack>
+	) : null;
+
+const SectionTitleContainer = ({ customFields }) => {
+	const { inheritGlobalContent = true, sectionContentConfig } = customFields;
```

Comment:
The related custom field in the prop types below should have a `defaultValue: true` as well, so the editor visual matches what the code is actually doing.

Reply:
@vgalatro Fixed